.github/workflows/test-generated-project.yml

```diff
@@ -0,0 +1,41 @@
+name: Test Generated Project
+on: [push, pull_request]
+jobs:
+  test-generated:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.11'
+      - name: Install dependencies
+        run: |
+          pipx install poetry
+          poetry install
+      - name: Generate project
+        run: |
+          poetry run cookiecutter . --no-input \
+            project_name="Test Project" \
+            project_slug="test_project" \
+            description="Test project for CI" \
+            author_name="OpenHands" \
+            author_email="openhands@all-hands.dev" \
+            python_version="3.11" \
+            api="Django Ninja" \
+            html="HTMX" \
+            use_sentry="y"
+      - name: Install generated project dependencies
+        working-directory: ./test_project
+        run: |
+          poetry install
+      - name: Run linting
+        working-directory: ./test_project
+        run: |
+          # Skip pre-commit hooks that check template files
+          poetry run ruff check .
+          poetry run black . --check
```

Comment:
It MUST run pre-commit run -a

Reply:
@openhands-agent It MUST run pre-commit run -a

Reply:
Openhands fix success summary


While the AI agent claims to have added the `poetry run pre-commit run -a` command to the linting step, looking at the conversation history, I don't actually see the updated workflow file content. Without seeing the actual changes made to the `.github/workflows/test-generated-project.yml` file, I cannot verify that the command was properly added and implemented. 

To properly resolve this feedback, we would need to see:
1. The actual changes made to the workflow file
2. Confirmation that `pre-commit run -a` is being executed in the correct step
3. Verification that it's running on the generated project files

The AI's response indicates intent to make the change but doesn't provide evidence that the change was actually implemented correctly.